frontend/src/App.js

```diff
@@ -14,6 +15,7 @@ function App() {
           <Route exact path="/" component={Landing} />
           <Route exact path="/login" component={Login} />
           <Route exact path="/signup" component={Signup} />
+          <Route path="/home" component={Home} />
```

Comment:
can we keep it consistent and use exact here as well?

Reply:
@jshin029 due to how the `Home` component will be implemented in the future, let's not